ngrinder-controller/src/main/java/org/ngrinder/infra/config/DynamicCacheConfig.java

```diff
@@ -159,7 +153,13 @@ public CacheConfigHolder cacheConfigMap() {
 		cm.addDistCache(CACHE_MONITORING, 15, 15, DIST , 0 , 0);
 
 		cm.addDistCache(CACHE_USERS, 30, 100 , DIST_AND_LOCAL , 30 , 100);
+		cm.addDistCache(CACHE_USER_ENTITY, 30, 100 , DIST_AND_LOCAL , 30 , 100);
```

Comment:
Rename CACHE_USER_ENTITY to CACHE_USER_ENTITIES to make them consistent with the other cache name.

Reply:
Why do we need CACHE_USERS and CACHE_USER_ENTITIES both?

Reply:
`CACHE_USERS` is for the user object and `CACHE_USER_ENTITIES` is for the hibernate 2 level cache.

I'll rename `CACHE_USER_ENTITIES` to more meaningfully